data/json/overmap_terrain.json

```diff
@@ -822,6 +822,16 @@
         "extras" : "field"
     },{
         "type" : "overmap_terrain",
+        "id" : "sugar_house",
+        "name" : "sugar house",
+        "sym" : 83,
+        "color" : "i_brown",
+        "see_cost" : 5,
+        "extras" : "build",
+        "mondensity" : 2,
+        "allow_road" : true
```

Comment:
In contrast, `"rotate" : true` should be present in `"overmap_terrain"` to enable rotation.
(this requirement is odd and inconsistent. I'm going to change it later).

I'd also remove `"allow_road" : true` to prevent overwriting it with a road.